### ISSUE TITLE
Search form color contrast

### ISSUE THREAD

**Issue by inghamn** [City-of-Bloomington/ureport-client#18]
*  The color contrast ratio between the search field border (#dee2e6) and its background (#ffffff) is 1.3:1. WCAG requires a minimum color contrast ratio of 3:1 between the field’s border and surrounding background. People with vision disabilities such as limited vision and color blindness may have difficulty seeing the input field.
*  The color contrast ratio of the search field border on focus (#86b7fe) and its background (#ffffff) is 2.06:1, which also fails WCAG.
